README.md

```diff
@@ -11,24 +11,24 @@ natural language processing.
 
 ## Abstract
 Chest X-Rays are considerably reliable radiobiological imprints of patients, which are widely used to diagnose an array 
-of common diseases affecting organs within the chest. For too long, vast accumulations of image data and their 
+of common diseases affecting organs within the chest. Over a period, vast accumulations of image data and their 
 associated diagnoses have been stored in the Picture Archiving and Communication Systems (PACS) of several hospitals and 
 medical institutions. 
 Meanwhile, data-hungry Deep Learning systems lie in wait of voluminous databases like these, 
 at the cusp of fulfilling the promise of automated and accurate disease diagnosis. 
 Through this project, we hope to unite one such vast database, the “ChestX-ray8" dataset, 
-with powerful Deep Learning Systems, and automate the diagnosis of 14 common lung conditions. 
-For this project phase, we focus on detecting three conditions - 
+with powerful Deep Learning System, and automate the diagnosis of 14 common lung conditions. 
+For this project phase, we focus on detecting three particluar conditions - 
 Cardiomegaly, Effusion, Emphysema - in addition to healthy states. 
 
 ## Introduction 
 Deep learning, also known as hierarchical learning or deep structured learning, 
 is a type of machine learning that uses a layered algorithmic architecture to analyze data. 
-Unlike other types of machine learning, deep learning has the added benefit of being able to make decisions with 
-significantly less involvement of human trainers. While basic machine learning requires a programmer to identify whether a conclusion 
+Unlike other types of machine learning, deep learning has an added advantage of being able to make decisions with 
+significantly less human intervention. While basic machine learning requires a programmer to identify whether a conclusion 
 is correct or not, deep learning can gauge the accuracy of its answers on its own due to the nature of its multi-layered structure. 
 The emergence of modern frameworks like PyTorch, has also made preprocessing of data more convenient. 
-Many of the filtering and normalization tasks that would have been completed manually by human programmers when 
+Many of the filtering and normalization tasks that would involve a lot of  human programmers when 
```

Comment:
Many of the filtering and normalization tasks  would involve a lot manual efforts while using other machine learning techniques, are taken up automatically.